packages/cli/generators/project/templates/package.plain.json.ejs

```diff
@@ -57,13 +57,13 @@
     "dist"
   ],
   "dependencies": {
-    "@loopback/context": ">=4.0.0-alpha.18",
+    "@loopback/context": ">=0.1.0",
 <% if (project.projectType === 'application') { -%>
-    "@loopback/core": ">=4.0.0-alpha.20",
-    "@loopback/rest": ">=4.0.0-alpha.7",
-    "@loopback/openapi-v2": ">=4.0.0-alpha.2"
+    "@loopback/core": ">=0.1.0",
+    "@loopback/rest": ">=0.1.0",
+    "@loopback/openapi-v2": ">=0.1.0"
```

Comment:
If we're switching to "real" semver values, should we switch back to caret-prefixed patterns (`^0.1.0`)?

Reply:
Good catch. Fixed.